### DISCUSSION TITLE
VS Code Server causes Z2M container to crash

### DISCUSSION THREAD

**Discussion by wildchai** [tteck/Proxmox#1805] [Help Needed]
Hi @tteck ,

Thanks again for your scripts. I just installed VS Code Server in my Z2M in an LXC, it works but any activity will gradually use up all the server resources and crash the container, might take from 1 min or 30 minutes before it crashes. Even closing the vscode page does not work. I will need to stop or reboot to release the resources.

![image](https://github.com/tteck/Proxmox/assets/140302945/1fcd6df9-d001-4769-8432-62b4f73592a5)

**Comment by tteck** [comment]
Try increasing memory

**Reply by wildchai** [reply]
I doubled up both memory and the swap. It still slowly ramps up continuously. Needed to close and open a new session before it hits red.
